pkg/operator/controller/vulnerabilityreport.go

```diff
@@ -108,6 +110,27 @@ func (r *VulnerabilityReportReconciler) reconcileWorkload(workloadKind kube.Kind
 			}
 		}
 
+		if r.Config.VulnerabilityScannerLatest && workloadKind == kube.KindReplicaSet {
+			controller := metav1.GetControllerOf(workloadObj)
+			if controller != nil && controller.Kind == string(kube.KindDeployment) {
+				deploymentPartialObject := kube.Object{
+					Kind:      kube.KindDeployment,
+					Name:      controller.Name,
+					Namespace: req.Namespace,
+				}
+				deploymentObject, err := r.GetObjectFromPartialObject(ctx, deploymentPartialObject)
```

Comment:
I think this requires get,list,watch deployments permission if you run it in cluster. Otherwise we'll be getting the following errors:

```
E1221 18:19:53.453587       1 reflector.go:138] pkg/mod/k8s.io/client-go@v0.22.4/tools/cache/reflector.go:167:
Failed to watch *v1.Deployment: failed to list *v1.Deployment: deployments.apps is forbidden:
User "system:serviceaccount:starboard-system:starboard-operator" cannot list resource "deployments"
in API group "apps" in the namespace "default"
```

Reply:
Ahh thought that it was already there and I did the old mistake and trying from go instead of running it using a image. I will fix it.